Makefile

```diff
@@ -58,7 +60,10 @@ $(GINKGO):
 	GOBIN="$(BIN_DIR)" go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.3
 
 $(KO):
-	@./scripts/download-ko.sh "$(BIN_DIR)"
+	@$(PROJECT_DIR)/scripts/download-ko.sh "$(BIN_DIR)"
+
+$(GUM):
+	@$(PROJECT_DIR)/scripts/download-gum.sh "$(BIN_DIR)"
```

Comment:
Will this download the Gum binary every time, or only if it does not already exist in `BIN_DIR`?